infrastructure/iac/iac.go

```diff
@@ -324,7 +338,7 @@ func (iac *Scanner) getExtendedMessage(issue iacIssue) string {
 
 }
 
-func (iac *Scanner) toIssue(affectedFilePath string, issue iacIssue, fileContent string) snyk.Issue {
+func (iac *Scanner) toIssue(affectedFilePath string, issue iacIssue, fileContent string) (*snyk.Issue, error) {
```

Comment:
Why did you change it to a pointer return value? Any reason?

Reply:
Changed in https://github.com/snyk/snyk-ls/pull/269/commits/d53881ee92568432ace2f313971cb3a49ed4c491?show-viewed-files=true&file-filters%5B%5D=